Remove log.warns (used for debugging) and some java doc errors

geowar1 opened the pull request.

geowar1 commented:
Travis has a COMPILATION ERROR : /home/travis/build/JMRI/JMRI/java/test/jmri/NamedBeanTest.java.
It's building just fine on my machine?!?

rhwood commented:
Try a clean and build (ant clean; ant debug) on your machine. You may be using a cached built copy of the file that Travis never starts with.

Randall Wood
Alexandria Software
randall.h.wood@alexandriasoftware.com
http://alexandriasoftware.com

> On May 25, 2017, at 15:45, George Warner <notifications@github.com> wrote:
> 
> Travis has a COMPILATION ERROR : /home/travis/build/JMRI/JMRI/java/test/jmri/NamedBeanTest.java.
> It's building just fine on my machine?!?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

rhwood commented:
Looking at your code, you merged in a file with conflicts and conflict markers.

Randall Wood
Alexandria Software
randall.h.wood@alexandriasoftware.com
http://alexandriasoftware.com

> On May 25, 2017, at 15:45, George Warner <notifications@github.com> wrote:
> 
> Travis has a COMPILATION ERROR : /home/travis/build/JMRI/JMRI/java/test/jmri/NamedBeanTest.java.
> It's building just fine on my machine?!?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

geowar1 commented:
My last commit shows those as having been removed… and `ant clean; ant debug` builds for me… GitHub Desktop isn't showing any changes since the last commit before the PR.

geowar1 commented:
I'll make a whitespace or comment change to <NamedBeanTest.java> and commit again…

geowar1 commented:
My bad; I looked at <`NamedBean.java`> instead of <`NamedBeanTest.java`>… (`ant clean; ant debug` doesn't build or run the test suite. So it doesn't catch build or run errors there. `ant alltest` would have.)

rhwood commented:
"ant tests" will build the tests without running them.

Randall Wood
Alexandria Software
randall.h.wood@alexandriasoftware.com
http://alexandriasoftware.com

> On May 25, 2017, at 16:06, George Warner <notifications@github.com> wrote:
> 
> My bad; I looked at <NamedBean.java> instead of <NamedBeanTest.java>… (ant clean; ant debug doesn't build or run the test suite. So it doesn't catch build or run errors there. ant alltest would have.)
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

geowar1 commented:
Passed! Merge please… TIA.
